content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -1,296 +1,425 @@
+2. At the bottom of the **Seed List files** panel, select the **Default Seed Lists** check box. Make sure to deselect the `GameSeedList` file.
 
-2. Click **Generate Asset Lists**. This will open the **Generate Asset List files** dialog. 
+3. Click **Generate Asset Lists**, which opens the **Generate Asset List files** dialog.
 
-3. In the list of platforms, select **pc** by enabling the check box.  
+4. In the list of platforms, select the **pc** check box.
 
-4. Click **Browse...** to open the File Explorer. 
+5. Click **Browse...**, which opens File Explorer.
 
-5. Enter a name for your asset list and save. In this example, we'll use the name `engine.assetlist`. 
+6. In File Explorer, enter a name for your asset list, and then click **Save**. For this example, we'll use the name `engine.assetlist`.
 
-5. Navigate to the **Asset Lists** tab to verify the assets in `engine.assetlist`. The assets are listed under the **Asset List** panel. 
+7. In the Asset Bundler, navigate to the **Asset Lists** tab to verify the assets in `engine.assetlist`. The assets are listed in the **Asset List** panel.
 
 #### Bundle your assets
 
-1. Select your `engine.assetlist` in the** Asset List Files** panel under the **Asset Lists** tab.
+1. In the Asset Bundler, on the **Asset Lists** tab, in the **Asset List Files** panel, select the `engine.assetlist` file.
+
+2. Click **Generate Bundle**, which opens the **Generate Bundles** dialog.
+
+3. Click **Browse...**, which opens File Explorer.
+
+4. In File Explorer, enter a name for your package file, and then click **Save**. For this example, use the name `engine.pak`.
+
+
+For Windows, when Asset Bundler saves your bundle, it appends `_pc` to the bundle's name. So, you now have two asset bundles: `game_pc.pak` and `engine_pc.pak`.
+
 
-2. Click **Generate Bundle**. This opens the **Generate Bundles** dialog. 
+### Add bundles to the project game release layout
 
-3. Click **Browse...** to open the File Explorer. 
 
-4. Enter a name for your package file and save. In this example, we'll use the name `engine_bundle.pak`.
+Next, add your `game_pc.pak` and `engine_pc.pak` files to your project game release layout, so the Game Launcher can load assets from those bundles.
 
+1. Navigate to the `<install>/bin/Windows/release/<build>/Cache/pc` directory, which contains the bundled content in your project game release layout directory. For example, the default path may be:
 
-You now have two bundled content: `game_bundle.pak` and `engine_bundle.pak`. 
+   - `C:/MyProject/install/bin/Windows/release/Default/Cache/pc` -- For non-monolithic.
 
+   - `C:/MyProject/install/bin/Windows/release/Monolithic/Cache/pc` -- For monolithic.
 
-### Add your bundles to install layout
+2. Remove the `engine.pak` file that was created earlier in [Create a project game release layout](#create-a-project-game-release-layout). You don't need it because you created new bundles.
```

Comment:
```suggestion
1. Remove the `engine.pak` file created earlier in [Create a project game release layout](#create-a-project-game-release-layout). You don't need it because you created new bundles.
```
Drop "that was" to avoid the obvious passive construction ("was created").